### PR TITLE
Migrate VGG16 to keras_cv.models

### PR DESCRIPTION
**Pull request by IMvision12**
Issue : #1159
Creates a keras_cv.model version of VGG16 ported from keras.applications.

@ianstenbit

### CODE REVIEW

File: keras_cv/models/vgg16_test.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+from keras_cv.models import vgg16
+
+from .models_test import ModelsTest
+
+MODEL_LIST = [
+    (vgg16.VGG16, 512, {}),
+]
+
+
+class VGG19Test(ModelsTest, tf.test.TestCase, parameterized.TestCase):
```
Comment (ianstenbit) [inline keras_cv/models/vgg16_test.py:27]: nit: VGG16Test

**Review by ianstenbit — APPROVED** [review]
One minor comment, then LGTM.

Thank you!

File: keras_cv/models/vgg16_test.py (hunk 1 above)
Comment (IMvision12) [reply keras_cv/models/vgg16_test.py]: Done !

**Comment by ianstenbit** [comment]
@IMvision12 it looks like the linter needs to be run on this PR -- can you please reformat?

**Comment by IMvision12** [comment]
@ianstenbit I have reformated the code using black

**Pull request merged**